src/Raven.Server/Smuggler/Documents/SmugglerExporter.cs

```diff
@@ -223,5 +218,21 @@ public ExportResult Export(DocumentsOperationContext context, Stream destination
             }
             return result;
         }
+
+        private void TimerCallback(object state)
```

Comment:
Bad idea, instead of doing it like this, have progress class hold the number of exported documents, and generate it on the fly when the client requesting it.